api/tools/src/main/resources/data_dictionary_exports/AllofUsDatasetV4_20201207.yaml

```diff
@@ -0,0 +1,7920 @@
+meta_data:
```

Comment:
How do I know which environment this file is for? In Preprod `All of Us Dataset v4` points to `R2019Q4R4` and in Prod `All of Us Dataset v4` points to `R2020Q4R2`.  These are 2 different data dictionaries? This seems problematic

Reply:
yeah.. IIRC this was because we didn't want to create an entry for every environment, CDR tuple so we needed a way for a single DD entry to refer to multiple CDRs. cdr_version_id didn't work since it's not the same across environments and I believe `bigquery_dataset` didn't work because multiple CDRs could refer to the same one in an environment.

The implementation for this feature was rushed out for a release and we never intended to keep this pattern of committing files to manage our data dictionaries. Maybe it's time we revisit and redesign the data ingestion portion. I'd love to be involved if we decide to do so.

Reply:
> I'd love to be involved if we decide to do so.

So this redesign feels like it should be part of our CDR indices build. This data dictionary is CDR driven and we have a different cloud mysql database that we build for all the data tools that is specific to a CDR. This really shouldn't exist outside of that context.